backends/http.go

```diff
@@ -37,12 +39,21 @@ func NewHTTP(authOpts map[string]string, logLevel log.Level) (HTTP, error) {
 
 	log.SetLevel(logLevel)
 
+	tr := &h.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
Whether peer verification is to be skipped should depend on the verifyPeer option.